deployment/aks-periscope.yaml

```diff
@@ -15,8 +15,8 @@ metadata:
   name: aks-periscope-role
 rules:
 - apiGroups: ["","metrics.k8s.io"]
-  resources: ["pods", "nodes"]
-  verbs: ["get", "watch", "list"]
+  resources: ["pods", "pods/portforward", "nodes"]
```

Comment:
just curious: what's this for?

Reply:
need this access to be able to run the `kubectl port-forward` command. Don't have the right permissions without it